var/spack/repos/builtin/packages/parallelmergetree/package.py

```diff
@@ -32,6 +42,8 @@ class Parallelmergetree(CMakePackage):
             tag='v1.0.0',
             submodules=True)
 
+    depends_on('babelflow@1.1.0', when='@1.1.2')
+    depends_on('babelflow@1.1.0', when='@1.1.1')
     depends_on('babelflow@1.1.0', when='@1.1.0')
```

Comment:
Would the following work instead of duplicating for each version:

```suggestion
    depends_on('babelflow@1.1.0', when='@1.1:')
```

Reply:
@sshudler @cyrush I'm going to skip this recommendation to get the package merged.   Do suggest this be revisited in another iteration on the package.